### PR TITLE
feat: Add geospatial post processing operations

### PR DESCRIPTION
**Pull request by villebro**
### CATEGORY

Choose one

- [ ] Bug Fix
- [x] Enhancement (new features, refinement)
- [ ] Refactor
- [x] Add tests
- [ ] Build / Development Environment
- [x] Documentation

### SUMMARY
Add geospatial post processing operations to new chart data endpoint.

### ADDITIONAL INFORMATION

- [x] Has associated issue: #9187
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Comment by villebro** [comment]
@ktmud in reference to your comment on #9427 , I'll add the rename change here.

**Comment by villebro** [comment]
@ktmud The change will require a slightly bigger refactor. I'll do this as a separate PR later to keep this PR simple. In the meantime you can do the select and rename by applying two consecutive selects; first renaming, next selecting. This will only cause negligible overhead, hence it might even be a better idea to split out all operations from `select` into separate operations to keep things simple: `select_columns`, `drop_columns` and `rename_columns`.

File: tests/pandas_postprocessing_tests.py [hunk 1]
```diff
@@ -288,3 +316,83 @@ def test_cum(self):
...
+        )
+        self.assertListEqual(
+            sorted(post_df.columns.tolist()),
+            sorted(["city", "geohash", "latitude", "longitude"]),
+        )
+        self.assertListEqual(
+            series_to_list(post_df["geohash"]), series_to_list(lonlat_df["geohash"]),
+        )
+
+    def geodetic_parse(self):
```
Comment (dpgaspar) [inline tests/pandas_postprocessing_tests.py:357]: rename to: `def test_geodetic_parse`

**Review by dpgaspar — APPROVED** [review]
LGTM, just a couple of non blocking comments

**Review by dpgaspar — CHANGES_REQUESTED** [review]
Actually, the test def name should be changed

File: superset/charts/schemas.py [hunk 2]
```diff
@@ -265,15 +265,23 @@ class ChartDataSelectOptionsSchema(ChartDataPostProcessingOperationOptionsSchema
     columns = fields.List(
         fields.String(),
         description="Columns which to select from the input data, in the desired "
-        "order. If columns are renamed, the old column name should be "
+        "order. If columns are renamed, the original column name should be "
         "referenced here.",
         example=["country", "gender", "age"],
+        required=False,
+    )
+    drop = fields.List(
```
[the remark this answers is not in the document]
Comment (villebro) [reply superset/charts/schemas.py]: Done. I was originally going to use `exclude`, but seems I forgot and ended up using `drop` as pandas uses that name.

File: tests/pandas_postprocessing_tests.py (hunk 1 above)
Comment (villebro) [reply tests/pandas_postprocessing_tests.py]: Good catch!

File: superset/utils/pandas_postprocessing.py [hunk 3]
```diff
@@ -325,23 +324,32 @@ def rolling(  # pylint: disable=too-many-arguments
 
 @validate_column_args("columns", "rename")
 def select(
-    df: DataFrame, columns: List[str], rename: Optional[Dict[str, str]] = None
+    df: DataFrame,
+    columns: Optional[List[str]] = None,
+    drop: Optional[List[str]] = None,
```
[the remark this answers is not in the document]
Comment (villebro) [reply superset/utils/pandas_postprocessing.py]: Good catch!

**Comment by villebro** [comment]
Thanks for the valuable comments @dpgaspar !

**Review by dpgaspar — APPROVED** [review]

**Pull request merged**